integration/presto/src/main/java/org/apache/carbondata/presto/CarbondataRecordSetProvider.java

```diff
@@ -73,27 +60,23 @@ public CarbondataRecordSetProvider(CarbondataConnectorId connectorId, CarbonTabl
 
     CarbondataSplit carbondataSplit =
         checkType(split, CarbondataSplit.class, "split is not class CarbondataSplit");
-    checkArgument(carbondataSplit.getConnectorId().equals(connectorId), "split is not for this connector");
+    checkArgument(carbondataSplit.getConnectorId().equals(connectorId),
+        "split is not for this connector");
 
-    StringBuffer targetColsBuffer = new StringBuffer();
     String targetCols = "";
     // Convert all columns handles
     ImmutableList.Builder<CarbondataColumnHandle> handles = ImmutableList.builder();
     for (ColumnHandle handle : columns) {
       handles.add(checkType(handle, CarbondataColumnHandle.class, "handle"));
-      targetColsBuffer.append(((CarbondataColumnHandle) handle).getColumnName()).append(",");
+      targetCols += ((CarbondataColumnHandle) handle).getColumnName() + ",";
```

Comment:
Actually last commit was not rebased with my previous one I will  correct it again